### DISCUSSION TITLE
`use_context` value can get lost when rendering a list of components

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#785] [Problem]
The title is fairly self-explanatory, but I haven't confirmed that's the exact cause. All I know is that I'm getting `None` when attempting to use `use_context` (via `django_idom.hooks.use_websocket`) for my list of sidebar tabs.

I think we should have a quick tag up for me to demo this issue. Since IDOM doesn't give a useful stack trace on exceptions like this, I'll need to walk you through the stack as well.

Let me know what your availability is.

**Comment by Archmonger** [comment]
This could also be due to conditional rendering (two return statements, one component).

I won't be available from the 13th to 20th of this month. Let me know if you have time before the 13th for me to demo.

**Reply by rmorshea** [reply]
Sunday morning or early afternoon would work for me again.
